src/main/java/jenkins/plugins/slack/SlackRequest.java

```diff
@@ -1,111 +1,9 @@
 package jenkins.plugins.slack;
 
-import java.util.Objects;
-import net.sf.json.JSONArray;
+import net.sf.json.JSONObject;
 
-public class SlackRequest {
-    private String message;
-    private String color;
-    private String timestamp;
-    private JSONArray attachments;
-    private JSONArray blocks;
+public interface SlackRequest {
```

Comment:
This has been gutted to just an interface implemented by `SlackMessageRequest` and `SlackReactionRequest`.